apps/ops/api/__init__.py

```diff
@@ -2,6 +2,7 @@
 #
 from .adhoc import *
 from .celery import *
+from .inventory import *
 from .job import *
 from .playbook import *
 from .variable import *
```

Comment:
The changes you suggested do not seem to be related to specific issues, regularities of deviations or optimizations. I didn't notice any major problems in this piece-wise code snippet so far, but it's always a good idea to double-check things if there is any uncertainty regarding the implementation.

If you're asking more about syntax corrections/improvements, please add those specifics.